### DISCUSSION TITLE
Replicating cilostazol beagle PK

### DISCUSSION THREAD

**Discussion by craigadunbar** [Open-Systems-Pharmacology/Forum#1824] [Questions & Problems]
I'm new to using PK-SIM and wanted to find my legs by replicating the cilostazol beagle PK simulation published by [Willman et al.](https://doi.org/10.1016/j.ejpb.2010.06.003). 

![Cilastozol beagle PK Willman](https://github.com/user-attachments/assets/897a3048-876e-4502-98de-7185bbd78508)

So far I've had mixed results and was hoping there may be a cilostazol repository somewhere in the archives (nothing available on GitHub). I think a cilostazol repository would provide a great way for onboarding new users to PK-SIM.

Below are the fasted and fed simulations I've generated for the three particle size distributions produced by hammer-mill, jet-mill, and nanocrystallization. Any help resolving the differences between these and the Willman simulations would be greatly appreciated.

### Fasted
![image](https://github.com/user-attachments/assets/fc8cf05a-a25e-4341-ab63-19b9bd17052a)

### Fed
![image](https://github.com/user-attachments/assets/ad42fd14-f6f2-42c0-9f99-848f95d67d88)

Cheers, Craig

**Comment by tobiasK2001** [comment]
Dear Craig,

Yes, it would be nice to have this model for onboarding new PK-Sim users. Unfortunately the model in the Willmann et al paper was built for PK-Sim 4.2. and there has been some major release changes between the current version. I guess it is not straightforward to replicate result in the current PK-Sim version. Maybe the authors @stefan.willmann@bayer.com, @Yuri05 can help here?

Best, Tobias

**Reply by craigadunbar** [reply]
Tobias

Thank you so much for highlighting this. I was aware of the version change from a single GI compartment (v.4.2) to multiple compartments (v.5 and beyond) as it was nicely described by the [Willman/Thelen/Lippert paper](https://doi.org/10.1111/j.2042-7158.2012.01534.x).

I'd be happy to share the cilostazol PK-SIM Project File I generated for anyone curious enough to dive in.

Cheers, Craig
